### ISSUE TITLE
Problem with "java.time.Duration" on Applovin adapter

### ISSUE THREAD

**Issue by sergip76** [cleveradssolutions/CAS-Android#15]
It is not possible to use the SDK with API lower than 26 due to this problem:

```
com.applovin.impl.l0.collectGoogleAdvertisingInfo
java.lang.ClassNotFoundException - Didn't find class "java.time.Duration" on path: DexPathList
```

It seems that the Applovin adapter uses the java.time.Duration class introduced in API 26.

**Comment by Str4tos** [comment]
Hi @sergip76, 
Thank you for creating the issue. 

Unfortunately, this part of the code is not related to the CAS AppLovin Adapter, but rather to the AppLovin SDK, which we cannot modify. However, your issue seems to be related to the Java version. Although Android API 26 introduced the `Duration` class natively, this class is also available starting from Java 1.8. 

Please make sure that your app is using at least Java 1.8. To do so, the following lines should be defined in your app-level `build.gradle` file:

```groovy
android {
    // ...
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }

    kotlinOptions {
        jvmTarget = '1.8'
    }
}
```

Let us know if you managed to resolve the issue.

**Comment by sergip76** [comment]
> Hi @sergip76, Thank you for creating the issue.
> 
> Unfortunately, this part of the code is not related to the CAS AppLovin Adapter, but rather to the AppLovin SDK, which we cannot modify. However, your issue seems to be related to the Java version. Although Android API 26 introduced the `Duration` class natively, this class is also available starting from Java 1.8.
> 
> Please make sure that your app is using at least Java 1.8. To do so, the following lines should be defined in your app-level `build.gradle` file:
> 
> ```groovy
> android {
>     // ...
>     compileOptions {
>         sourceCompatibility JavaVersion.VERSION_1_8
>         targetCompatibility JavaVersion.VERSION_1_8
>     }
> 
>     kotlinOptions {
>         jvmTarget = '1.8'
>     }
> }
> ```
> 
> Let us know if you managed to resolve the issue.

Yes. I was already using Java 1.8 with the code you wrote. It seems it doesn't solve the problem.

**Comment by balakumar-t** [comment]
@sergip76 - Same reported to Applovin, you may get fix soon.

https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/issues/465

**Comment by sergip76** [comment]
> @sergip76 - Same reported to Applovin, you may get fix soon.
> 
> [AppLovin/AppLovin-MAX-Unity-Plugin#465](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/issues/465)

Thank you very much @balakumar-t. I hope Applovin will fix it quickly.

**Comment by ucf0709** [comment]
Still  having this issue in my apps
